pnpm-workspace.yaml

```
@@ -1,3 +1,4 @@
 packages:
-  - packages/*
-  - e2e
+  - 'packages/*'
+  - 'e2e'
+  - '!packages/http-client-csharp/*'
```

```suggestion
  - '!packages/http-client-csharp/**'
```

When I was testing the ** seemed to be required for this pattern.